src/content/contributing/website/style_guide.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "Docs Style Guide"
+date: 2020-04-29T19:09:55+02:00
+weight: 5
+---
+
+## Documentation Style Guide
+
+This guide is meant to help keep our documentation consistent and ease the
+contribution and review process.
+
+### Submariner project and binary capitalization
+
+The project name "Submariner" should be capitalized. The same goes for
+Submariner subprojects like Lighthouse, Shipyard, Coastguard and similar.
+
+To indicate that a Submariner project artifact should be taken as the literal
+characters of a command, Submariner project binaries like `subctl` can be
+written with \`backticks\`.
+
```

Comment:
I think you can include a section with a note on K8s objects (like Pod/Service/Deployment etc) as we use them frequently in our documentation.

Reply:
Nice suggestion @sridhargaddam, thanks! While starting to add this I was digging around and noticed that the K8s style guide covers it nicely in their docs about object capitalization. Maybe I can re-work this PR to first say we follow the K8s Style Guide (which would cover this rule), and extend parts of it (like their [word list](https://kubernetes.io/docs/contribute/style/style-guide/#kubernetes-io-word-list)) to add Submariner-specific things.

Reply:
Done in the latest. :)